inference-engine/src/inference_engine/CMakeLists.txt

```diff
@@ -161,7 +161,7 @@ if (TBBBIND_2_4_FOUND)
 endif()
 
 target_link_libraries(${TARGET_NAME} PRIVATE pugixml::static openvino::itt ${CMAKE_DL_LIBS} Threads::Threads
-                                             ngraph ngraph::frontend_manager inference_engine_transformations)
+                                             ngraph ngraph::frontend_manager::static inference_engine_transformations)
```

Comment:
Why do we need this `::static` postfix?

Reply:
In this case it will look similar to pugixml::static (see line above)